src/Elastic.Documentation.Site/Assets/web-components/SearchOrAskAi/Search/SearchResults/SearchResultsListItem.tsx

```diff
@@ -126,25 +131,36 @@ export function SearchResultListItem({
                     /* Focus ring for selected and focus states */
                     &[data-selected],
                     &:focus {
-                        outline: 2px solid ${euiTheme.colors.primary};
-                        outline-offset: -2px;
+                        outline: 1px solid ${euiTheme.colors.borderBasePlain};
```

Comment:
I'm a bit unsure about this.

The default focus styling suddenly disappears when tabbing through the elements.

Also, I don't think that the hovered and selected state should look the same.

https://github.com/user-attachments/assets/7086e240-1767-419d-b84d-3bb20be54c09

We might need to rework this and change the implementation and **not** use focus for navigating through the items.

This way, the focus style could still apply when using tabs, but a different styling when using arrow keys.